exec/jdbc/src/main/java/org/apache/drill/jdbc/impl/DrillDatabaseMetaDataImpl.java

```diff
@@ -54,6 +55,10 @@ private void throwIfClosed() throws AlreadyClosedSqlException,
     }
   }
 
+  private RpcEndpointInfos getServerInfos() throws SQLException {
+    DrillConnectionImpl connection = (DrillConnectionImpl) getConnection();
+    return connection.getClient().getServerInfos();
```

Comment:
yes, because the client is initialized as part of the constructor of DrillConnectionImpl (the field is final too)